src/network/network.h

```diff
@@ -59,36 +59,38 @@ namespace Network {
   public:
     string function;
     int the_errno;
+
   private:
     string my_what;
+
   public:
-    NetworkException( string s_function="<none>", int s_errno=0)
-      : function( s_function ), the_errno( s_errno ),
-        my_what(function + ": " + strerror(the_errno)) {}
-    const char *what() const throw () { return my_what.c_str(); }
-    ~NetworkException() throw () {}
+    NetworkException( string s_function = "<none>", int s_errno = 0 )
+      : function( s_function ),
+        the_errno( s_errno ),
+        my_what( function + ": " + strerror( the_errno ) ) {}
+    const char *what() const throw() { return my_what.c_str(); }
+    ~NetworkException() throw() {}
   };
 
-  enum Direction {
-    TO_SERVER = 0,
-    TO_CLIENT = 1
-  };
+  enum Direction { TO_SERVER = 0, TO_CLIENT = 1 };
```

Comment:
I think these should be newline separated

Reply:
Ditto.